webapp/lib/model.dart

```diff
@@ -44,7 +44,57 @@ extension ConversationUtil on g.Conversation {
   }
 }
 
-UnmodifiableListView<g.Tag> tagIdsToTags(Iterable<String> tagIds, Iterable<g.Tag> allTags) {
+extension MessageUtil on g.Message {
+  /// Add [tagId] to tagIds in this Message.
+  /// Callers should catch and handle IOException.
+  Future<void> addTagId(g.DocPubSubUpdate pubSubClient,
+      g.Conversation conversation, String tagId) async {
```

Comment:
This is more of a style question, but I find the split on two lines here a bit hard to read (same for the other methods below). Is this because of a linter being a bit eager on line length? Would it be possible to disable that check? thanks!

Reply:
Good point. I'll work on readability for these three methods in a subsequent CL.

Reply:
Fixed in https://github.com/larksystems/nook/pull/222